inspirehep/base/searchext/mappings/hep.json

```diff
@@ -78,28 +95,79 @@
                         "properties": {
                             "value": {
                                 "type": "string",
-                                "index": "not_analyzed"
+                                "index": "not_analyzed",
+                                "copy_to": ["doi"]
```

Comment:
We have to make sure, at record creation/modification, that the DOI is normalized e.g. to `10.xxx/xxx`, without any superflous prefix, and that the search term is receiving the same treatment. Alternatively we need, for every DOI to index it also with `doi:` and `http://dx.doi.org/` prefixes. This seems something that could be made easily with a Groovy script.

https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-scripting.html